src/main/java/qupath/ext/align/gui/ImageAlignmentPane.java

```diff
@@ -370,10 +370,7 @@ public ImageAlignmentPane(final QuPathGUI qupath) {
 		Button btnReset = new Button("Reset");
 		btnReset.setOnAction(e -> {
 			var overlay = getSelectedOverlay();
-			var affine = overlay == null ? null : overlay.getAffine();
-			if (affine == null)
-				return;
-			affine.setToIdentity();
+			overlay.resetAffine();
```

Comment:
Is the null check not needed, i.e. is there absolutely no way for `getSelectedOverlay()` to return null?

Reply:
Thank you for pointing this out Pete, 

> Is the null check not needed, i.e. is there absolutely no way for `getSelectedOverlay()` to return null?

I honestly don't know so I've restored the check on `overlay == null` you had in place previously (commit https://github.com/qupath/qupath-extension-align/pull/10/commits/a662e99d096c15cb59112d87333cef33b80b1146)